web_widget_kanban_many2one/static/src/xml/web_widget_kanban_many2one.xml

```diff
@@ -0,0 +1,9 @@
+<template>
+    <t t-name="Kanban_editable_temp">
+        <span class="container">
+            <select class="form-control text-success selectM2MField" name="id">
```

Comment:
This "form-control" class induces the CSS issues that we talked about - either add some CSS to counter it, or lose the class here